homeworks/irina.vlasova_IrinaBondarenko/homework_6/defineProperty.js

```diff
@@ -0,0 +1,26 @@
+const user = {};
+Object.defineProperty(user, 'fullName', {
+    configurable: true,
+    enumerable: true,
+    set: (name) => {
+        const a = name.replace(/([A-Za-z])/g, s => s.toLowerCase());
+        this.fullName = a.replace(/(^|\s|[-])\S/g, s => s.toUpperCase());
```

Comment:
defining a setter with arrow function will effect global scope

also, try to use less heavy regular expression check, you can look for something like `\b\w`